rust/processor/src/utils/counters.rs

```diff
@@ -228,9 +228,9 @@ pub static TRANSACTION_UNIX_TIMESTAMP: Lazy<GaugeVec> = Lazy::new(|| {
 });
 
 /// Data gap warnings
-pub static PROCESSOR_DATA_GAP_COUNT: Lazy<IntCounterVec> = Lazy::new(|| {
-    register_int_counter_vec!("indexer_processor_data_gap_count", "Data gap count", &[
-        "type"
+pub static PROCESSOR_DATA_GAP_COUNT: Lazy<IntGaugeVec> = Lazy::new(|| {
```

Comment:
we can actually delete this metric, just checked it's not used anywhere

Reply:
I added a use for it in this PR 😬